packages/lit-element/src/lib/lit-element.ts

```diff
@@ -240,54 +222,39 @@ export class LitElement extends UpdatingElement {
   }
 
   /**
-   * Applies styling to the element shadowRoot using the [[`styles`]]
-   * property. Styling will apply using `shadowRoot.adoptedStyleSheets` where
-   * available and will fallback otherwise. When Shadow DOM is polyfilled,
-   * ShadyCSS scopes styles and adds them to the document. When Shadow DOM
-   * is available but `adoptedStyleSheets` is not, styles are appended to the
-   * end of the `shadowRoot` to [mimic spec
-   * behavior](https://wicg.github.io/construct-stylesheets/#using-constructed-stylesheets).
+   * Applies the given styles to the element. Styling is applied to the element
+   * only if the `renderRoot` is a `shadowRoot`. If the `rendeRoot` is not
+   * a `shadowRoot`, this method may be overridden to apply styling in another
+   * way. Styling will apply using `shadowRoot.adoptedStyleSheets` where
+   * available and will fallback otherwise. When Shadow DOM is available but
+   * `adoptedStyleSheets` is not, styles are appended to the the `shadowRoot`
+   * to [mimic spec behavior](https://wicg.github.io/construct-stylesheets/#using-constructed-stylesheets).
    */
-  protected adoptStyles() {
-    const styles = (this.constructor as typeof LitElement)._styles!;
-    if (styles.length === 0) {
+  protected adoptStyles(styles: CSSResultArray) {
+    // Note, if renderRoot is not a shadowRoot, styles would/could apply to the
+    // element's getRootNode(). While this could be done, we're choosing not to
+    // support this now since it would require different logic around de-duping.
+    if (!(this.renderRoot instanceof window.ShadowRoot)) {
       return;
     }
-    // There are three separate cases here based on Shadow DOM support.
-    // (1) shadowRoot polyfilled: use ShadyCSS
-    // (2) shadowRoot.adoptedStyleSheets available: use it
-    // (3) shadowRoot.adoptedStyleSheets polyfilled: append styles after
-    // rendering
-    if (window.ShadyCSS !== undefined && !window.ShadyCSS.nativeShadow) {
-      window.ShadyCSS.ScopingShim!.prepareAdoptedCssText(
-        styles.map((s) => (s as CSSResult).cssText),
-        this.localName
-      );
-    } else if (supportsAdoptingStyleSheets) {
+    if (supportsAdoptingStyleSheets) {
       (this.renderRoot as ShadowRoot).adoptedStyleSheets = styles.map((s) =>
-        s instanceof CSSStyleSheet ? s : s.styleSheet!
+        s instanceof CSSStyleSheet ? s : (s as CSSResult).styleSheet!
```

Comment:
Is the `CSSResult.styleSheet` only called when guarded by a `if (supportsAdoptingStyleSheets)` check? If so, you can remove the check in the `styleSheet` getter.

Reply:
Done.